base/standard/src/main/java/org/openscience/cdk/tools/manipulator/AtomContainerManipulator.java

```diff
@@ -1541,15 +1545,15 @@ public static void percieveAtomTypesAndConfigureUnsetProperties(IAtomContainer c
 
     /**
      * This method will reset all atom configuration to UNSET.
-     *
+     * <br>
```

Comment:
In HTML terms, you should use `<p>` here, I think

Reply:
I usually use both, depending on whether I want a paragraph or just a line break.

Happy to change to a paragraph if that is what you prefer.

Reply:
This is not a breaking issue for me :)